modules/installation-ibm-power-vs-config-yaml.adoc

```diff
@@ -87,11 +91,12 @@ If you disable simultaneous multithreading, ensure that your capacity planning a
 //====
 //<6> You can optionally provide the `sshKey` value that you use to access the machines in your cluster.
 //endif::openshift-origin[]
-<4> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<5> The name of an existing resource group.
-<6> Required. The installation program prompts you for this value.
+<4> The SMTLevel specifies the level of SMT to set to the control plane and compute machines. The supported values are from 1 to 8.
```

Comment:
You cannot set it to any value between 1 and 8.  For e.g. 3, 5, 6, 7 etc.  Only Powers of 2 are allowed, 1, 2, 4, 8.